test/sslapitest.c

```diff
@@ -1899,17 +1971,22 @@ static int ocsp_client_cb(SSL *s, void *arg)
 static int test_tlsext_status_type(void)
 {
     SSL_CTX *cctx = NULL, *sctx = NULL;
+    char *leaf_chain = test_mk_file_path(certsdir, "leaf-chain.pem");
+    char *skey = test_mk_file_path(certsdir, "leaf.key");
+    char *leaf = test_mk_file_path(certsdir, "leaf.pem");
     SSL *clientssl = NULL, *serverssl = NULL;
     int testresult = 0;
     STACK_OF(OCSP_RESPID) *ids = NULL;
     OCSP_RESPID *id = NULL;
     BIO *certbio = NULL;
+    OSSL_LIB_CTX *tmpctx = OSSL_LIB_CTX_set0_default(libctx);
 
     if (!create_ssl_ctx_pair(libctx, TLS_server_method(), TLS_client_method(),
                              TLS1_VERSION, 0,
-                             &sctx, &cctx, cert, privkey))
+                             &sctx, &cctx, leaf, skey))
         return 0;
```

Comment:
Returning here leaks `leaf` and `skey`.
```suggestion
        goto end;
```

Reply:
I've done it: https://github.com/openssl/openssl/pull/28105